pages/dkp/kommander/1.4/clusters/index.md

```diff
@@ -7,70 +7,87 @@ menuWeight: 7
 excerpt: View clusters created with Kommander or any connected Kubernetes cluster
 ---
 
-Kommander allows you to monitor and manage very large numbers of clusters, so it's easy to either connect existing clusters or create new clusters whose lifecycle is managed by Konvoy.
+Kommander allows you to monitor and manage very large numbers of clusters. Use the features described in this area of the product to connect existing clusters, or to create new clusters whose lifecycle is managed by Konvoy. You can view clusters from the Clusters tab in the navigation pane on the left. You can see the details for a cluster by selecting the **View Details** link at the bottom of the cluster card or the cluster name in either the card or the table view.
 
 ## Types
 
-There are several types of clusters to be aware of in the Clusters tab.
+There are several types of clusters that display in the Clusters tab. The cluster type appears in the cluster card just under the cluster name.
 
-- **Attached**: A cluster that was not created with Kommander. Attached clusters' lifecycle cannot be managed.
-- **Managed**: A Konvoy cluster that was created with Kommander. Managed clusters' lifecycle can be managed.
-- **Management**: The Konvoy cluster that hosts Kommander.
+![Cluster card with types highlighted](/dkp/kommander/1.4/img/cluster-card-types.png)
```

Comment:
@JohnB64-JBV - this image is not uploaded yet (but that might be a known thing thus far)

Reply:
Should be there now.